crates/core/src/lib.rs

```diff
@@ -37,15 +45,15 @@ pub unsafe extern "C" fn compile_src(js_src_ptr: *const u8, js_src_len: usize) -
     // Use fresh runtime to avoid depending on Wizened runtime
     let runtime = runtime::new_runtime().unwrap();
     let js_src = str::from_utf8(slice::from_raw_parts(js_src_ptr, js_src_len)).unwrap();
+
     let bytecode = runtime
-        .context()
-        .compile_module(FUNCTION_MODULE_NAME, js_src)
+        .compile_to_bytecode(FUNCTION_MODULE_NAME, js_src)
         .unwrap();
-    let bytecode_len = bytecode.len();
+
     // We need the bytecode buffer to live longer than this function so it can be read from memory
-    let bytecode_ptr = Box::leak(bytecode.into_boxed_slice()).as_ptr();
+    let bytecode_ptr = Box::leak(bytecode.clone().into_boxed_slice()).as_ptr();
```

Comment:
Nit: I'm interested in why this was changed to introduce a clone. It works so it's fine.

Reply:
Good catch, yeah. I've restored the previous implementation to prevent that clone. The clone was required because was I was using `bytecode.len()` after calling `into_boxed_slice`.